bauble/src/types.rs

```diff
@@ -99,21 +111,31 @@ pub struct TypeRegistry {
     primitive_types: [TypeId; 5],
 }
 
+#[allow(missing_docs)]
 pub enum VariantKind {
     Flattened(TypeKind),
     Explicit(Fields),
 }
 
+/// A variant usable by an enum in Bauble.
```

Comment:
Probably worth mentioning that this is only used to pass to `TypeRegistry::build_enum`